### DISCUSSION TITLE
moving folders

### DISCUSSION THREAD

**Discussion by kim-fox** [requarks/wiki#6676] [Help / Questions]
salute
please can someone help me, I'm having issues about moving folders, not page, but oflders from one oint to another, I'd like to move a subfolder and is subfolders outside of a folder and alocate it in another folder, please I've already tried even the cls

**Comment by aagha** [comment]
Everything is paths and files, so you need to change the path

**Reply by kim-fox** [reply]
okay, for example 
<img width="337" alt="pag1" src="https://github.com/requarks/wiki/assets/23312748/9a07511b-3224-4351-bdef-7041a01f9303">
in the first image I've moved the page outside linux directory,  but in second image, look at path, the directory troubleshoot didn't move outside of linux directory, only the page
<img width="495" alt="pag2" src="https://github.com/requarks/wiki/assets/23312748/575831ea-4173-4dfa-b4b0-3acd221b2d83">

**Reply by docdoceg** [reply]
It's because you have other pages with paths that go through /troubleshoot. You need to update the path for all pages that lead to/through that directory for it to go away.
